internal/provider/resource_storagezone.go

```diff
@@ -26,6 +26,8 @@ const (
 	keyRewrite404To200    = "rewrite_404_to_200"
 )
 
+	var RegionException		= []string{"SYD", "SG", "BR"}
```

Comment:
1. RegionException should be a private variable (lowercase).
2. I  suggest to rename it to something like:
```suggestion
var storageZoneRegionsRequiringReplication = []string{"SYD", "SG", "BR"}
```
to make it clear from the name for what it is.